common-rest/src/main/java/org/uniprot/api/rest/output/context/MessageConverterContext.java

```diff
@@ -41,7 +41,7 @@ public class MessageConverterContext<T> {
     private boolean isLargeDownload;
     private boolean subsequence;
     /**
-     * A map to store accessions along with their associated sequence ranges and a flag indicating
+     * A mapto to store accessions along with their associated sequence ranges and a flag indicating
      * if they have been processed. Accessions can be repeated, allowing multiple sequence ranges
```

Comment:
Please be careful with global find and replace. Please revert this and similar change map is indeed needed.